tests/by-util/test_watch.rs

```diff
@@ -10,3 +10,44 @@ use crate::common::util::TestScenario;
 fn test_invalid_arg() {
     new_ucmd!().arg("--definitely-invalid").fails().code_is(1);
 }
+
+#[test]
+fn test_invalid_interval() {
+    let args = vec!["-n", "definitely-not-valid", "true"];
+    new_ucmd!().args(&args).fails();
```

Comment:
could you please check stderr too here?
(at least with .contains, doesn't have to be a full match)

Reply:
Added in latest commit